lib/slime/compiler.ex

```diff
@@ -5,27 +5,51 @@ defmodule Slime.Compiler do
 
   alias Slime.Doctype
 
-  alias Slime.Parser.Nodes.{DoctypeNode, EExNode, HTMLCommentNode, HTMLNode, InlineHTMLNode, VerbatimTextNode}
+  alias Slime.Parser.Nodes.{DoctypeNode, EExNode, HEExNode, HTMLCommentNode, HTMLNode, InlineHTMLNode, VerbatimTextNode}
+
+  alias Slime.TemplateSyntaxError
+
+  @eex_delimiters {"#" <> "{", "}"}
+  @heex_delimiters {"{", "}"}
```

Comment:
As far as I can tell, the values of these globals are never used, except for comparing if the passed "delimiters" parameter matches one or the other.

Given this, perhaps it would be better to pass a symbol instead? Something like `complile(x, :eex)` or `compile(x, %{output_format: :eex})`?